R/impact_central.R

```diff
@@ -152,22 +152,16 @@ impact_by_year_of_vaccination <- function(meta1, raw_impact, fvps, fvps_updates
     fvps_updates <- fvps_updates[fvps_updates$year %in% vaccination_years, ]
   }
 
-
+  raw_impact$birth_cohort <- raw_impact$time
```

Comment:
In the new functions I'm using `birth_cohort` as the column name instead of `time` which gives this awkward renaming step.

I see Xiang has it hardcoded to check that `time` column exists above https://github.com/vimc/vimpact/pull/7/files#diff-94b59a1086fde6063424f28c1f56d3bbR113 I'm happy to drop this change and just use `time` everywhere if that is a term that is familiar to science as representing `birth_cohort` otherwise I think it is better to be more specific in the naming

Reply:
I like the switch to birth_cohort